ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/radiobutton/v1/radiobutton/clientlibs/site/js/radiobuttonview.js

```diff
@@ -107,6 +107,16 @@
             });
         }
 
+        #syncWidget() {
+            let widgetElementList =this.getWidget();
+            if(widgetElementList){
+                widgetElementList.forEach((widgetElement)=>{
+                    let newId=this.getId()+"__widget"+widgetElement.id.substring(widgetElement.id.lastIndexOf('_'))
```

Comment:
spaces between operators pls to make it readable.

Reply:
radiobutton.html can be overlaid and may give rise to completely new d format. lets have it in the customisation docs to override the method for syncing markup.

Reply:
> spaces between operators pls, to make it readable.

Done